plugin/hover.py

```diff
@@ -174,21 +175,27 @@ def _show_hover(self, point: int) -> None:
                     location=point,
                     wrapper_class=css().popups_classname,
                     max_width=800,
-                    on_navigate=lambda href: self.on_hover_navigate(href, point))
+                    on_navigate=lambda href: self._on_navigate(href, point))
 
-    def on_hover_navigate(self, href: str, point: int) -> None:
-        if href.startswith('subl:'):
+    def _on_navigate(self, href: str, point: int) -> None:
+        if href.startswith("subl:"):
             pass
-        elif href.startswith('code-actions'):
+        elif href.startswith('code-actions:'):
             _, config_name = href.split(":")
             titles = [command["title"] for command in self._actions_by_config[config_name]]
             sel = self.view.sel()
             sel.clear()
             sel.add(sublime.Region(point, point))
-
             self.view.show_popup_menu(titles, lambda i: self.handle_code_action_select(config_name, i))
+        elif href.startswith("file://"):
+            window = self.view.window()
+            if window:
+                window.open_file(href[len("file://"):], flags=sublime.ENCODED_POSITION)
         else:
-            webbrowser.open_new_tab(href)
+            if not (href.lower().startswith("http://") or href.lower().startswith("https://")):
+                href = "http://" + href
```

Comment:
I don't like that too much. Assuming `http://` might actually not work. If it's due to removing protocol in frontmatter then I would probably prefer not removing protocol in the first place.

(But you can decide on that)

Reply:
Based on this comment I played around with disabling magiclink and/or hide_protocol, but the problem remains: `webbrowser.open` for some reason doesn't open `github.com`, but it does open `http://github.com` and/or `https://github.com`.

Reply:
hide_protocol turns `http://github.com` into `[github.com](http://github.com)` by the way, so it's not a problem.

Reply:
Oh, then it's fine. We'll only potentially fail for something that didn't have the protocol in the first place.

Reply:
Yeah so the py3.3 `webbrowser` module seems broken with respect to not doing anything when http:// is absent, but the py3.8 `webbrowser` module does work. One more reason to move to py3.8 :)